_search-plugins/knn/api.md

```diff
@@ -291,32 +334,28 @@ DELETE /_plugins/_knn/models/{model_id}
 }
 ```
 
-## Train Model
-Introduced 1.2
-{: .label .label-purple }
+## Train model
 
-Create and train a model that can be used for initializing k-NN native library indexes during indexing. This API will 
-pull training data from a `knn_vector` field in a training index and then create and train a model and then serialize it
-to the model system index. Training data must match the dimension passed into the body of the request. This request 
-will return when training begins. To monitor the state of the model, use the [Get model API](#get-model).  
+Create and train a model that can be used for initializing k-NN native library indexes during indexing. This API pulls training data from a `knn_vector` field in a training index, creates and trains a model, and then serializes it to the model system index. Training data must match the dimension passed in the request body. This request returns when training begins. To monitor the model's state, use the [Get model API](#get-model).  
 
-Query Parameter |  Description
+Query parameter |  Description
 :--- | :---
-`model_id` | (Optional) The id of the fetched model. If not specified, a random id will be generated.
-`node_id` | (Optional) Preferred node to execute training. If set, this node will be used to perform training if it is deemed to be capable.
+`model_id` | (Optional) The unique identifier of the fetched model. If not specified, a random id is generated.
+`node_id` | (Optional) Specifies the preferred node to execute the training process. If provided, the specified node is used for training if it has the necessary capabilities and resources available.
 
-Request Parameter |  Description
+Request parameter |  Description
 :--- | :---
-`training_index` | Index from where training data from.
-`training_field` | `knn_vector` field from `training_index` to grab training data from. Dimension of this field must match `dimension` passed in to this request.  
-`dimension` | Dimension this model is for.
-`max_training_vector_count` | (Optional) Maximum number of vectors from the training index to use for training. Defaults to all of the vectors in the index.
-`search_size` | (Optional) Training data is pulled from the training index with scroll queries. Defines the number of results to return per scroll query. Defaults to 10,000.
-`description` | (Optional) User provided description of the model.
-`method` | Configuration of ANN method used for search. For more information about possible methods, refer to the [method documentation]({{site.url}}{{site.baseurl}}/search-plugins/knn/knn-index#method-definitions). Method must require training to be valid.
+`training_index` | The index from which the training data is obtained.
+`training_field` | The `knn_vector` field in the `training_index` from which the training data is retrieved. The dimension of this field must match `dimension` passed in this request.  
```

Comment:
Nit: We use "retrieved" here but "obtained" in the previous line.

Reply:
Revised to use "retrieved"